source/data-api/generated-endpoints.txt

```diff
@@ -570,15 +559,39 @@ specified in your :ref:`Data API configuration <data-api-response-type>`.
 Authenticate the Request
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
-Incoming requests must include headers that authenticate the calling
-user.
+If an endpoint is configured to use Application Authentication then you
+must include a valid user access token or login credentials with every
+request.
+
+In general, prefer to use an access token over login credentials when
+possible. The token lets you run multiple requests without
+re-authenticating the user. It also lets you send requests from a web
+browser or other environment that enforces :wikipedia:`CORS
+<Cross-origin_resource_sharing>`.
 
-Requests can authenticate with any of the following authentication
-providers by including user credentials directly in the request headers:
+To use an access token, authenticate the user through an authentication
+provider and then include the access token in the request's
+Authorization header using a Bearer token scheme. For more information
+on how to acquire and use an access token, see :ref:`Bearer Token
+Authentication <graphql-bearer-authentication>`.
```

Comment:
not sure why GH won't let me leave a suggestion here, but the ref is wrong.

https://github.com/mongodb/docs-app-services/blob/14d7b153e39f3d78baa77cfed511a35492966d2c/source/data-api/generated-endpoints.txt#L576

i think should be `data-api-bearer-authentication`

Reply:
d'oh that was a brainfart - fixed!